CadRevealFbxProvider/Attributes/ScaffoldingAttributeParser.cs

```diff
@@ -22,7 +55,7 @@ private static string ConvertStringToEmptyIfNullOrWhiteSpace(string? s)
     public (
         Dictionary<string, Dictionary<string, string>?> attributesDictionary,
         ScaffoldingMetadata scaffoldingMetadata
-    ) ParseAttributes(string[] fileLines)
+    ) ParseAttributes(string[] fileLines, bool tempFlag = false)
```

Comment:
This method is a bit long. Maybe some of the functionallity could be split out into separate methods?
E.g.
```
var attributeRawData = GetAttributeRawData(fileLines);
var keyIdColumn = GetKeyIdColumn(attributeRawData);
var validatedAttributeData = GetValidatedAttributeData(attributeRawData, keyIdColumn);

var entireScaffoldingMetadata = new ScaffoldingMetadata();
var attributesDictionary = validatedAttributeData.ToDictionary(...
```

Maybe the code inside the ToDictionary could split up some into separate methods as well?